packages/avatar/src/Avatar.js

```diff
@@ -165,14 +222,31 @@ class Avatar extends Component {
 
   render() {
     const { size, name, ...otherProps } = this.props;
+    const { firstName, lastName } = otherProps;
```

Comment:
I think these should come from `this.props` above rather than `otherProps` since these are actual declared props for the component. Also `className` is already being destructured from `otherProps`.

Reply:
Moved `firstName` and `lastName` declarations to `this.props` destructuring